MiYoga/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+<header>
+    <div id="logodark"></div>
+    <nav>
+        <ul>
+            <li>Our Team</li>
+            <li>Classes</li>
+            <li>Tutorials</li>
+            <li>Contacts</li>
+
+        </ul>
+    </nav>
+    <div id="button"></div>
+    <div id="massYoga"><h1>Mass Yoga</h1></div>
+    <div id="miYogaisyo">
+        <p>
+            MI Yoga is your number one source for in-depth yoga pose instruction, yoga sequences for beginners to advanced practitioners
+        </p>
+    </div>
+</header>
+
+<section id="hero-image">
+    <div id="image-banner">
+        <div id="image-button1">
+        <div id="description">
+            <p id="vid">Video Tutorial</p>
+            <p>Harness the power of your dreams</p>
+        </div>
+        <div id="play"></div>
+
+    </div>
+
+    </div>
+    <div id="discover"><p>Discover more</p></div>
+    <div id="discoverline"><p></p></div>
+    <div id="discoverpage"><p>1</p></div>
+    <div id="discoverlin2"><p></p></div>
+    <div id="discoverpage2"><p>4</p></div
+</section>
+
+<section id="sequences">
+    <div id="seq-dec">
+        <div class="short"><div class="line"></div>Why choose MI yoga</div>
+        <div id="text">Yoga<br><span class="highlight">Sequences for</span> Beginners</div>

```

Comment:
Lyginant su originaliuoju dizainu, klausimas, kodėl class "highlight" priskirta "Sequences for", o ne "for Beginners"?